indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -63,6 +63,8 @@
   </div>
   % endif
 
+  ${ template_hook('survey-announcement', event=conf) }
```

Comment:
Ahh, ok. I see the point right now, because you can do sth like:
`@template_hook('announcement'):
def fun():
    pass

@template_hook('announcement'):
def fun():
    pass'
I thought you can associate a hook with only one function.